tests/test_mdlp.py

```diff
@@ -32,165 +34,31 @@ def test_find_cut_no_cut():
-        [ 2.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  1.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  1.,  1.,  2.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  0.,  2.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  1.,  1.,  1.],
-        [ 2.,  1.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  0.,  1.,  1.],
-        [ 0.,  0.,  1.,  1.],
-        [ 1.,  0.,  1.,  1.],
-        [ 2.,  1.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 0.,  0.,  1.,  1.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  1.,  2.,  2.],
-        [ 1.,  0.,  1.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  2.,  1.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  1.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 1.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.],
-        [ 2.,  1.,  2.,  2.],
-        [ 2.,  0.,  2.,  2.]]
+  # discretization is invariant under translation of the data
+  translated_disc = MDLP(shuffle=False).fit_transform(X - 5, y)
```

Comment:
This is nice to test! Can you put it in another test case though?

Reply:
yup.